### ISSUE TITLE
Multiline string literals get wrapped in unnecessary parens

### ISSUE THREAD

**Issue by JelleZijlstra** [ambv/black#215]
Operating system: OS X
Python version: 3.7
Black version: master

Our codebase like to write code like
```python
x = """
a really long string
"""
```
(For example, for storing SQL queries.) Black turns this into
```python
x = (
    """
a really long string
"""
)
```
The parens don't do anything useful here.

**Comment by ambv** [comment]
Good catch! This is because `right_hand_split()` tries to omit the parentheses and discovers that the line is still "too long" so it leaves them in.

The easiest workaround here will be to just avoid adding optional parentheses to multiline strings in the first place since they are always considered "too long".

I'll work on this for the beta.
